src/SIL.XForge.Identity/Controllers/Account/AccountController.cs

```diff
@@ -284,6 +382,15 @@ private async Task<LoginViewModel> BuildLoginViewModelAsync(LoginInputModel mode
             return vm;
         }
 
+        private ForgotPasswordViewModel BuildForgotPasswordViewModelAsync()
```

Comment:
Since this is no longer an async method, you can rename this to `BuildForgotPasswordViewModel`.

Reply:
Yes, we removed async.